genie-test/src/main/java/com/netflix/genie/test/suppliers/RandomSuppliers.java

```diff
@@ -33,16 +34,15 @@
      * Get a random String.
      */
     public static final Supplier<String> STRING = UUID.randomUUID()::toString;
+    private static final Random RANDOM = new Random();
```

Comment:
Spotbugs complained about inefficiencies of the other implementation so switched to this.